src/lib/progress.c

```diff
@@ -113,7 +113,7 @@ void progress_report(double count, double max)
 			if (progress_function) {
 				progress_function(step.description, step.current, step.total, percentage);
 			} else {
-				info("\r\t...%d%%", percentage);
+				info("\t...%d%%\r", percentage);
```

Comment:
Instead of printing this here I would move that to the else of the if (!isatty) so we wont print it to files